tests/suites/spaces_ec2/task.sh

```diff
@@ -0,0 +1,27 @@
+test_spaces_ec2() {
+    if [[ "$(skip 'test_spaces_ec2')" ]]; then
+        echo "==> TEST SKIPPED: space tests (EC2)"
+        return
+    fi
+    if [[ "$BOOTSTRAP_PROVIDER" != "aws" ]]; then
```

Comment:
We need to update `main.sh` to sniff the cloud from `juju show-controller` when using an already bootstrapped controller and export BOOTSTRAP_PROVIDER...

Reply:
https://github.com/juju/juju/pull/10862/commits/176c41e5f73709778f2802e4afefb89b0a41c1b0